### ISSUE TITLE
missing migrations?!?

### ISSUE THREAD

**Issue by jedie** [aldryn/aldryn-forms#147]
Seems that there are missing migrations, e.g.:

```
Migrations for 'aldryn_forms':
  .../lib/python3.5/site-packages/aldryn_forms/migrations/0012_auto_20180206_1533.py
    - Alter field action_backend on formplugin
Migrations for 'email_notifications':
  .../lib/python3.5/site-packages/aldryn_forms/contrib/email_notifications/migrations/0003_auto_20180206_1533.py
    - Change Meta options on emailnotification
```

**Comment by czpython** [comment]
Fixed by https://github.com/aldryn/aldryn-forms/pull/148

**Comment by wfehr** [comment]
I just installed version 3.0.3 and django tells me that the migration (0012) is missing (same migration content as in #154 ).
I have django-cms 3.5.2 + django 1.11.12 installed.

**Comment by jeroenpeters1986** [comment]
I have the same issue as @wfehr 
`./manage.py migrate` informs me about that, I have installed:
`django-cms==3.5.1` and `Django==1.11.13`

**Comment by czpython** [comment]
@jeroenpeters1986 can you post a gist of the resulting migration?

**Comment by jeroenpeters1986** [comment]
@czpython yes, sure!

It's the migration named _Alter field action_backend on formplugin_

```
# -*- coding: utf-8 -*-
# Generated by Django 1.11.13 on 2018-07-24 08:31
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('aldryn_forms', '0011_auto_20180110_1300'),
    ]

    operations = [
        migrations.AlterField(
            model_name='formplugin',
            name='action_backend',
            field=models.CharField(choices=[(b'email_only', 'Email only'), (b'none', 'None'), (b'default', 'Default')], default=b'default', max_length=15, verbose_name='Action backend'),
        ),
    ]

```

**Comment by czpython** [comment]
Thanks @jeroenpeters1986 
Can you try the 3.0.4 release?

**Comment by wfehr** [comment]
@czpython Installed version 3.0.4, but I am still getting the same migration as @jeroenpeters1986 (django-cms==3.5.2, Django==1.11.12).

**Comment by czpython** [comment]
@wfehr with the binary strings?

**Comment by jeroenpeters1986** [comment]
@czpython , yes that fixes it! I'm not entirely sure, but I believe this happened because there were fixes done in 3.0.3 while it was already released and we had it installed

Installing 3.0.4 fixes it and does not give notices about not all migrations applied
